apps/fluent-tester/src/TestComponents/Switch/Switch.tsx

```diff
@@ -0,0 +1,113 @@
+import * as React from 'react';
+import { Test, TestSection, PlatformStatus } from '../Test';
+import { SWITCH_TESTPAGE } from './consts';
+import { View, StyleSheet } from 'react-native';
+import { Switch } from '@fluentui-react-native/switch';
+import { E2ESwitchTest } from './E2ESwitchTest';
+import { commonTestStyles } from '../Common/styles';
+
+const styles = StyleSheet.create({
+  square: {
+    backgroundColor: 'red',
+    width: 100,
+    height: 100,
+  },
+});
+
+const StandardUsage: React.FunctionComponent = () => {
+  return (
+    <View style={commonTestStyles.settingsPicker}>
+      <Switch defaultChecked={true} label={'Default Checked True'} />
+      <Switch defaultChecked={false} label={'Default Checked False'} />
+      <Switch defaultChecked={true} label={'Disabled Default Checked True'} disabled />
+      <Switch defaultChecked={false} label={'Disabled Default Checked False'} disabled />
+    </View>
+  );
+};
+
+const OnChangeUsage: React.FunctionComponent = () => {
+  const [displaySquare, setDisplaySquare] = React.useState(true);
+  const [evenDisplaySquare, setEvenDisplaySquare] = React.useState(true);
+  const [count, setCount] = React.useState(0);
+
+  const defaultToggleSquare = (checked: boolean) => {
+    setDisplaySquare(checked);
+  };
+
+  const toggleSquareEven = (checked: boolean) => {
+    if (count % 2 == 1) {
```

Comment:
nit: the even-ness could probably be represented with a boolean instead of a count.

```
let isEven = false;

if (checked) {
    isEven = !isEven;
    setEvenDisplaySquare(isEven);    
} else {
    setEvenDisplaySquare(false);    
}
```

Reply:
I actually removed this example and replaced it with something else. I was trying to show how you could control the checked state of the Switch. But I realize a much simpler and more understandable example would be to just have 2 buttons that change the Switch's state. So that's what I've added to replace this instead.